base-notebook/start.sh

```diff
@@ -54,7 +54,8 @@ if [ "$(id -u)" == 0 ] ; then
         # (it could be mounted, and we shouldn't create it if it already exists)
         if [[ ! -e "/home/${NB_USER}" ]]; then
             echo "Relocating home dir to /home/${NB_USER}"
```

Comment:
```suggestion
            echo "Copying home dir to /home/${NB_USER}"
```

This code is quite scary in general for anyone that has mounted something, which is what this PR has concluded and is mitigating.

I'm not feeling confident to say `mv`, `cp`, or `ln` is an acceptable practice. Perhaps `cp` is better than `mv` but yikes this feels very dangerous no matter what.

Reply:
I agree this looks scary.
Is there any better way though? Especially if we don't want to break existing users.

Reply:
Yeah its a bad situation no matter what is done I feel.

This will incur a performance hit as well now that didn't happen before as well, and its all hidden behind this layer of scripts doing things for users in a way that few really understands - even among us maintainers.

I'm inclined to think about removing a lot of scripts entirely and making what remain very stringently defined, tested, and documented.